templates/content/GeoBlazor.Template.WebAssembly/README.md

```diff
@@ -0,0 +1,12 @@
+TODO:
+
+If you are using an ApiKey supplied from the server add the following to your appsettings.json file in wwwroot:
+```
+{
+  "ArcGISApiKey": "YourAPIKey"
+} 
+```
+Otherwise, you can delete the appsettings.json file in wwwroot and remove the following line from the program.cs file
+```
+builder.Services.AddSingleton<IConfiguration>(_ => builder.Configuration);
```

Comment:
I feel like there might be other patterns in GB that use the `IConfiguration`?

Reply:
right now its the only one. For WebAssembly, it was not added by default, and right now it only reads the ArcGisToken